file2.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE users(
+	username       TEXT PRIMARY KEY NOT NULL,
+	email          TEXT    NOT NULL,
```

Comment:
Oh, one thing I think we should do is make emails case insensitive in our database, since emails are case insensitive.